File: web/js/tarjeta.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
!(function (a) {
    a(function () {
        a(".button-sent #back").hide(),
                a(".button-sent #continue").click(function (b) {
            a("#area .master-card").css("transform", "rotateY(180deg)"),
                    a(".button-sent #back").show();
        }),
                a(".button-sent #back").click(function (b) {
            a("#area .master-card").css("transform", "rotateY(0deg)"),
                    a(this).hide();
        });
    });
})(jQuery);

File: web/Carpetas/Clases/Tarjeta.java
package Carpetas.Clases;
import java.util.Date;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author Dheler!
 */
public class Tarjeta {
    private int id_tarjeta;
    private int num_tarjeta;
    private Date fecha_expiracion;

    public Tarjeta(int id_tarjeta, int num_tarjeta, Date fecha_expiracion){
    this.id_tarjeta = id_tarjeta;
    this.num_tarjeta= num_tarjeta;
    this.fecha_expiracion= fecha_expiracion;
    }
        
     public Tarjeta(){
         
     }
    public int getId_tarjeta() {
        return id_tarjeta;
    }

    public void setId_tarjeta(int id_tarjeta) {
        this.id_tarjeta = id_tarjeta;
    }

    public int getNum_tarjeta() {
        return num_tarjeta;
    }

    public void setNum_tarjeta(int num_tarjeta) {
        this.num_tarjeta = num_tarjeta;
    }

    public Date getFecha_expiracion() {
        return fecha_expiracion;
    }

    public void setFecha_expiracion(Date fecha_expiracion) {
        this.fecha_expiracion = fecha_expiracion;
    }
    
    
}
